### ISSUE TITLE
Bug: Chairs on escape shuttle face the wrong way

### ISSUE THREAD

**Issue by DooblyNoobly** [unitystation/unitystation#1237]
# Description

When the shuttle is facing right for some reason the chairs are facing up

**Comment by DooblyNoobly** [comment]
Leaving this for anyone else who attempts to fix this bug.: 

So it turns out all shuttles are set to Orientation.Up on initialOrientation state. This is a problem because the escape shuttle is mapped in an Orientation.Right direction. Changing initial state to right makes the shuttle turn to the right pointing the mapped sprite downwards. 

To overcome this I subtracted the initialOrientation.Degrees from any Orientation move methods.

See the gist below for how far I got with debugging: https://gist.github.com/DooblyNoobly/ff377c2935ad2da6b2919d00225f4ebd

^ this works but when the shuttle has to reverse in to dock it all goes wrong and the shuttle gets stuck. So maybe my solution wasn't a good one and there might be a better way to solve this.
